locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+msgstr "Resetar todos os dados"
+
+#: src/screens/LockedWallet.js:97
+msgid "Unlock"
+msgstr "Desbloquear"
+
+#: src/screens/NewWallet.js:154
+msgid ". A new wallet is generated by 24 words."
+msgstr ". Uma nova wallet é gerada a partir de 24 palavras."
+
+#: src/screens/NewWallet.js:155
+msgid ""
+". To have access to this wallet you must have all the words saved in the "
+"same order we will show to you."
+msgstr ""
+". Para recuperar o acesso a sua wallet, você deve salvar essas 24 palavras "
+"na mesma ordem que nós iremos te mostrar."
+
+#: src/screens/NewWallet.js:156
+msgid ""
+". If someone manages to discover your words they can steal your tokens, so "
+"we advise you to save your words physically and don't show them to anyone."
+msgstr ""
+". Se alguém tiver acesso as suas palavras, seus tokens podem ser roubados. "
+"Por isso, nós recomendamos que você salve suas palavras e não as mostre pra "
+"ninguém."
+
+#: src/screens/NewWallet.js:160
+msgid "Ok, I got it!"
+msgstr "Ok, eu entendi!"
+
+#: src/screens/NewWallet.js:165
+msgid "Create my words"
+msgstr "Criar minhas palavras"
+
+#: src/screens/NewWallet.js:174
+msgid "Your words have been created!"
+msgstr "Suas palavras foram criadas com sucesso!"
+
+#: src/screens/NewWallet.js:175
+msgid ""
+"You should save them in a non-digital media, such as a piece of paper. We "
+"advise you to do it now, but you can do it later."
+msgstr ""
+"You deve salvá-las em uma mídia não digital, como um pedaço de papel. Nós "
+"recomendamos que você faça isso agora, mas você pode fazer depois se desejar."
+
+#: src/components/ModalBackupWords.js:225 src/screens/NewWallet.js:178
+msgid "Do it later"
+msgstr "Fazer depois"
+
+#: src/screens/NewWallet.js:179
+msgid "Backup now"
+msgstr "Fazer o backup agora"
+
+#: src/screens/NewWallet.js:208 src/screens/Wallet.js:102
+msgid "Backup completed!"
+msgstr "Backup completo!"
+
+#: src/screens/Page404.js:27
+msgid "Page not found"
+msgstr "Página não encontrada"
+
+#: src/screens/Page404.js:29
+msgid "You tried to access a page that does not exist in Hathor Wallet"
+msgstr "Você tentou acessar uma página que não existe na Hathor Wallet"
+
+#: src/screens/PushTx.js:70
+msgid "Force push"
+msgstr "Forçar o envio"
+
+#: src/screens/PushTx.js:78
+msgid "Push tx"
+msgstr "Enviar transação"
+
+#: src/screens/PushTx.js:78
+msgid "decode"
+msgstr "decodificar"
+
+#: src/screens/PushTx.js:78
+msgid ""
+"Write your transaction in hex value and click the button to send it to the "
+"network. (We do not push blocks to the network, only transactions)"
+msgstr ""
+"Escreva sua transação em hexadecimal e clique no botão para enviá-la para a "
+"rede. (Não é possível enviar blocos, apenas transações)"
+
+#: src/screens/PushTx.js:80
+msgid "Transaction pushed to the network with success!"
+msgstr "Transação enviada para a rede com sucesso!"
+
+#: src/screens/SendTokens.js:143
+msgid "All your tokens were already added"
+msgstr "Todos os seus tokens já foram adicionados"
```

Comment:
"Todos os seus tokens já haviam sido adicionados"?  Ou a frase original era pra ter outro sentido?

Reply:
It's the same meaning. If you have 3 tokens in your wallet and want to add them in a transaction you receive "All your tokens were already added" when clicking "Add another token"

Reply:
Different meaning I think. "All your tokens were already added" sounds like nothing was done because the tokens you were trying to add were added before, so when you tried adding them again they were already added. If that's not what the original phrase meant we could fix it, maybe "Your tokens have been added" or "The selected tokens have been added", or "All  your tokens have been added" (if it really means that __all__ your tokens are being added), or maybe "included" instead of "added".

Reply:
Also, sometimes it isn't clear to me if "token" refers to the type of token (HTR) or the amount of tokens (like 10.00). I don't know if this is because I'm reading these out of their contexts or if there aren't any good words for those.